docker-compose/apps/middleman/.env.sample

```
@@ -0,0 +1,44 @@
+# Compose
+COMPOSE_PROJECT_NAME=igniter-middleman
+
+# Nextjs
+NODE_ENV="production"
+LOG_LEVEL="info"
+
+# Temporal.io
+TEMPORAL_URL="temporal:7233"
+TEMPORAL_NAMESPACE="middleman"
+TEMPORAL_TASK_QUEUE="middleman-operations"
+TEMPORAL_WORKFLOW_RETENTION="604800"
+
+# Postgresql
+PGHOST="postgresql"
+PGUSER="igniter"
+# CHANGE THIS (this should be the same of POSTGRES_PASSWORD at dependencies/.env)
+PGPASSWORD="igniter"
+DATABASE_URL="postgres://igniter:igniter@postgresql:5432/middleman?sslmode=disable"
+DB_NAME="middleman"
+
+# Used on worker/workflows to deliver txs
+# Set the proper one for the right network. You can use your own or one of the public ones:
+# https://dev.poktroll.com/category/explorers-faucets-wallets-and-more
+POKT_RPC_URL="https://shannon-testnet-grove-rpc.beta.poktroll.com"
+
+PROVIDERS_CDN_URL: "https://soothe-vault.nyc3.cdn.digitaloceanspaces.com/{chainId}/providers.json"
```

Shouldn't this be the real github URL?

```suggestion
PROVIDERS_CDN_URL: "https://raw.githubusercontent.com/stake-igniter/governace/refs/heads/main/{chainId}/provider.json"
```